app/controllers/auth_token_controller.rb

```diff
@@ -0,0 +1,27 @@
+class AuthTokenController < ApplicationController
```

Comment:
Let's drop a high-level comment? Like "subclass this controller to require "X-API-AUTH-TOKEN" header with value in `#config_auth_token`"?

Reply:
👍🏼